clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/model/EcsInstance.java

```diff
@@ -0,0 +1,82 @@
+package com.netflix.spinnaker.clouddriver.ecs.model;
+
+import com.amazonaws.services.ec2.model.InstanceStatus;
+import com.amazonaws.services.ecs.model.Task;
+import com.netflix.spinnaker.clouddriver.ecs.EcsCloudProvider;
+import com.netflix.spinnaker.clouddriver.model.HealthState;
+import com.netflix.spinnaker.clouddriver.model.Instance;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+public class EcsInstance implements Instance, Serializable {
+  private String name;
+  private HealthState healthState;
+  private Long launchTime;
+  private String zone;
+  private List<Map<String, String>> health;
+  private String providerType;
+  private String cloudProvider;
+
+  public EcsInstance(String name, Task task, InstanceStatus ec2Instance) {
+    this.name = name;
+    providerType = cloudProvider = EcsCloudProvider.ID;
+    launchTime = task.getStartedAt().getTime();
+    health =  null;
```

Comment:
You can either assign null to health on line 18, or instantiate an empty map on line 18.  Or on this line.  But assigning null to health here is a bit confusing

Reply:
The reasoning behind setting it up this way is that the expectation is that the constructor will assign a meaninful value. But since that part is not yet implemented, we are just assigning a null value. 

I do not feel strongly about it. I can change it if you do.

Reply:
I don't feel strongly about it either.  Let's move on